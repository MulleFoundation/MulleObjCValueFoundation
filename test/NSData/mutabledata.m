#ifndef __MULLE_OBJC__
# import <Foundation/Foundation.h>
#else
# import <MulleObjCValueFoundation/MulleObjCValueFoundation.h>
#endif


MULLE_OBJC_GLOBAL
void   MulleObjCHTMLDumpUniverse( void);


int   main( void)
{
   id   value;

   value = [NSData data];
   mulle_printf( "NSData -mulleIsThreadSafe: %btd\n", [value mulleIsThreadSafe]);

   value = [NSMutableData data];
   mulle_printf( "NSMutableData -mulleIsThreadSafe: %btd\n", [value mulleIsThreadSafe]);

   MulleObjCHTMLDumpUniverse();

   return( 0);
}
