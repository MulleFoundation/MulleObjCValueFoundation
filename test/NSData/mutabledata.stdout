NSData -mulleIsThreadSafe: YES
NSMutableData -mulleIsThreadSafe: NO
